samples/python/chat_sample/README.md

```diff
@@ -1,6 +1,18 @@
 # Python chat_sample that supports most popular models like LLaMA 3
 
-This example showcases inference of text-generation Large Language Models (LLMs): `chatglm`, `LLaMA`, `Qwen` and other models with the same signature. The application doesn't have many configuration options to encourage the reader to explore and modify the source code. For example, change the device for inference to GPU. The sample fearures `openvino_genai.LLMPipeline` and configures it for the chat scenario. There is also a Jupyter [notebook](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/llm-chatbot) which provides an example of LLM-powered Chatbot in Python.
+This example showcases the inference of text-generation Large Language Models (LLMs): `chatglm`, `LLaMA`, `Qwen`, and other models with the same signature. The application has a few configuration options, and we encourage the reader to explore and modify the source code to experiment with. For example, we can utilize the GPU for inference by changing the device name. 
+
+```
+device = 'CPU'  # GPU can be used as well
+pipe = openvino_genai.LLMPipeline(args.model_dir, device)
```

Comment:
If you believe that should be shown in readme, you must do that for every sample.